weld/resources/merger/init_merger.ll

```diff
@@ -0,0 +1,19 @@
+  ; Copy the initial value into the first merger
```

Comment:
Yes; this is similar to how the `vecmerger` is done (I'd like to change all the large LLVM strings in this file to use this templating method at some point).